views/lobby.ejs

```diff
@@ -192,6 +192,10 @@
 								<br>
 							
 								<button id="playerHighlightColourButton">Click to change highlight colour</button>
+								<br>
+								<button id="setHighlightColorToYellowButton" onclick="setAllHighlightColorsToYellow()">Set all highlight colors to yellow</button>
+								<br>
+								<button id="removeHighlight" onclick="unhighlightAllPlayers()">Unhighlight all players</button>
```

Comment:
Perhaps call unhighlightAllPlayers() when you leave a room instead of as a button, probably don't need such a button for this feature.

Reply:
Sometimes in the 8-10p game you might have 7+ players highlighted, then in the middle of the game 5+ of them unmerlin...So it's easier to unhighlight all of them and highlight just 2 you want than leaving the room or manually unhighlight 6-7 of them

Reply:
Well I dont think they'd all unmerlin at the same time, but if you want it could you space the buttons out a bit then? A simple `<br>` would probably do the trick. Also put it below the toggleable menu that "Click to change highlight colour" button toggles.